Compile error on aarch64 in version 0.8.4

```
error[E0423]: expected function, found macro `transmute`
   --> /Users/purewhite/.cargo/registry/src/rsproxy.cn-0dccff568467c15b/ahash-0.8.4/src/operations.rs:128:15
    |
128 |         xor ^ transmute::<_, u128>(vaesmcq_u8(vaeseq_u8(value, transmute!(0u128))))
    |               ^^^^^^^^^^^^^^^^^^^^ not a function
    |
help: use `!` to invoke the macro
    |
128 |         xor ^ transmute::<_, u128>!(vaesmcq_u8(vaeseq_u8(value, transmute!(0u128))))
    |                                   +
help: consider importing one of these items instead
    |
1   + use core::mem::transmute;
    |
1   + use std::mem::transmute;
    |
1   + use zerocopy::macro_util::core_reexport::mem::transmute;
    |

error[E0423]: expected function, found macro `transmute`
   --> /Users/purewhite/.cargo/registry/src/rsproxy.cn-0dccff568467c15b/ahash-0.8.4/src/operations.rs:161:15
    |
161 |         xor ^ transmute::<_, u128>(vaesimcq_u8(vaesdq_u8(value, transmute!(0u128))))
    |               ^^^^^^^^^^^^^^^^^^^^ not a function
    |
help: use `!` to invoke the macro
    |
161 |         xor ^ transmute::<_, u128>!(vaesimcq_u8(vaesdq_u8(value, transmute!(0u128))))
    |                                   +
help: consider importing one of these items instead
    |
1   + use core::mem::transmute;
    |
1   + use std::mem::transmute;
    |
1   + use zerocopy::macro_util::core_reexport::mem::transmute;
    |

For more information about this error, try `rustc --explain E0423`.
error: could not compile `ahash` (lib) due to 2 previous errors
```

env:

```
rustc 1.75.0-nightly (42b1224e9 2023-10-15)
macOS 14.0 with Apple Silicon
```

--- a/.github/workflows/rust.yml
+++ b/.github/workflows/rust.yml
@@ -91,7 +91,7 @@ jobs:
           args: --target armv7-unknown-linux-gnueabihf
   aarch64-apple-darwin:
     name: Aarch64 Apple Darwin
-    runs-on: ubuntu-latest
+    runs-on: macos-latest
     steps:
       - uses: actions/checkout@v2
       - uses: actions-rs/toolchain@v1

--- a/.github/workflows/rust.yml
+++ b/.github/workflows/rust.yml
@@ -51,7 +51,7 @@ jobs:
     env:
         RUSTFLAGS: -C target-cpu=native
     steps:
-      - uses: actions/checkout@v2 
+      - uses: actions/checkout@v2
       - name: Install latest nightly
         uses: actions-rs/toolchain@v1
         with:
@@ -89,6 +89,19 @@ jobs:
         with:
           command: check
           args: --target armv7-unknown-linux-gnueabihf
+  aarch64-apple-darwin:
+    name: Linux ARMv7
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          target: aarch64-apple-darwin
+      - uses: actions-rs/cargo@v1
+        with:
+          command: check
+          args: --target aarch64-apple-darwin
   i686-unknown-linux-gnu:
     name: Linux i686
     runs-on: ubuntu-latest

--- a/.github/workflows/rust.yml
+++ b/.github/workflows/rust.yml
@@ -90,7 +90,7 @@ jobs:
           command: check
           args: --target armv7-unknown-linux-gnueabihf
   aarch64-apple-darwin:
-    name: Linux ARMv7
+    name: Aarch64 Apple Darwin
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "ahash"
-version = "0.8.4"
+version = "0.8.5"
 authors = ["Tom Kaitchuck <Tom.Kaitchuck@gmail.com>"]
 license = "MIT OR Apache-2.0"
 description = "A non-cryptographic hash function using AES-NI for high performance"

--- a/src/operations.rs
+++ b/src/operations.rs
@@ -123,7 +123,6 @@ pub(crate) fn aesenc(value: u128, xor: u128) -> u128 {
     use core::arch::aarch64::*;
     #[cfg(target_arch = "arm")]
     use core::arch::arm::*;
-
     let value = transmute!(value);
     let res = unsafe { vaesmcq_u8(vaeseq_u8(value, transmute!(0u128))) };
     let value: u128 = transmute!(res);

--- a/src/operations.rs
+++ b/src/operations.rs
@@ -155,8 +155,7 @@ pub(crate) fn aesdec(value: u128, xor: u128) -> u128 {
     use core::arch::aarch64::*;
     #[cfg(target_arch = "arm")]
     use core::arch::arm::*;
-    let value = transmute!(value);
-    let res = unsafe { vaesimcq_u8(vaesdq_u8(value, transmute!(0u128))) };
+    let res = unsafe { vaesimcq_u8(vaesdq_u8(transmute!(value), transmute!(0u128))) };
     let value: u128 = transmute!(res);
     xor ^ value
 }

--- a/src/operations.rs
+++ b/src/operations.rs
@@ -124,12 +124,10 @@ pub(crate) fn aesenc(value: u128, xor: u128) -> u128 {
     #[cfg(target_arch = "arm")]
     use core::arch::arm::*;
 
-    unsafe {
-        let value = transmute!(value);
-        let res = unsafe { vaesmcq_u8(vaeseq_u8(value, transmute!(0u128))) };
-        let value: u128 = transmute!(res);
-        xor ^ value
-    }
+    let value = transmute!(value);
+    let res = unsafe { vaesmcq_u8(vaeseq_u8(value, transmute!(0u128))) };
+    let value: u128 = transmute!(res);
+    xor ^ value
 }
 
 #[cfg(all(any(target_arch = "x86", target_arch = "x86_64"), target_feature = "aes", not(miri)))]

--- a/src/operations.rs
+++ b/src/operations.rs
@@ -126,7 +126,10 @@ pub(crate) fn aesenc(value: u128, xor: u128) -> u128 {
     use core::mem::transmute;
     unsafe {
         let value = transmute!(value);
-        xor ^ transmute::<_, u128>(vaesmcq_u8(vaeseq_u8(value, transmute!(0u128))))
+    let value = transmute!(value);
+    let res = unsafe { vaesmcq_u8(vaeseq_u8(value, transmute!(0u128))) };
+    let value: u128 = transmute!(res);
+    xor ^ value
     }
 }
 
@@ -157,10 +160,10 @@ pub(crate) fn aesdec(value: u128, xor: u128) -> u128 {
     use core::arch::aarch64::*;
     #[cfg(target_arch = "arm")]
     use core::arch::arm::*;
-    use core::mem::transmute;
-    unsafe {
-        let value = transmute!(value);
-        xor ^ transmute::<_, u128>(vaesimcq_u8(vaesdq_u8(value, transmute!(0u128))))
+    let value = transmute!(value);
+    let res = unsafe { vaesimcq_u8(vaesdq_u8(value, transmute!(0u128))) };
+    let value: u128 = transmute!(res);
+    xor ^ value
     }
 }
 

--- a/src/operations.rs
+++ b/src/operations.rs
@@ -123,8 +123,7 @@ pub(crate) fn aesenc(value: u128, xor: u128) -> u128 {
     use core::arch::aarch64::*;
     #[cfg(target_arch = "arm")]
     use core::arch::arm::*;
-    let value = transmute!(value);
-    let res = unsafe { vaesmcq_u8(vaeseq_u8(value, transmute!(0u128))) };
+    let res = unsafe { vaesmcq_u8(vaeseq_u8(transmute!(value), transmute!(0u128))) };
     let value: u128 = transmute!(res);
     xor ^ value
 }

--- a/src/operations.rs
+++ b/src/operations.rs
@@ -123,6 +123,7 @@ pub(crate) fn aesenc(value: u128, xor: u128) -> u128 {
     use core::arch::aarch64::*;
     #[cfg(target_arch = "arm")]
     use core::arch::arm::*;
+    use core::mem::transmute;
     unsafe {
         let value = transmute!(value);
         xor ^ transmute::<_, u128>(vaesmcq_u8(vaeseq_u8(value, transmute!(0u128))))
@@ -156,6 +157,7 @@ pub(crate) fn aesdec(value: u128, xor: u128) -> u128 {
     use core::arch::aarch64::*;
     #[cfg(target_arch = "arm")]
     use core::arch::arm::*;
+    use core::mem::transmute;
     unsafe {
         let value = transmute!(value);
         xor ^ transmute::<_, u128>(vaesimcq_u8(vaesdq_u8(value, transmute!(0u128))))

--- a/src/operations.rs
+++ b/src/operations.rs
@@ -123,13 +123,12 @@ pub(crate) fn aesenc(value: u128, xor: u128) -> u128 {
     use core::arch::aarch64::*;
     #[cfg(target_arch = "arm")]
     use core::arch::arm::*;
-    use core::mem::transmute;
+
     unsafe {
         let value = transmute!(value);
-    let value = transmute!(value);
-    let res = unsafe { vaesmcq_u8(vaeseq_u8(value, transmute!(0u128))) };
-    let value: u128 = transmute!(res);
-    xor ^ value
+        let res = unsafe { vaesmcq_u8(vaeseq_u8(value, transmute!(0u128))) };
+        let value: u128 = transmute!(res);
+        xor ^ value
     }
 }
 
@@ -164,7 +163,6 @@ pub(crate) fn aesdec(value: u128, xor: u128) -> u128 {
     let res = unsafe { vaesimcq_u8(vaesdq_u8(value, transmute!(0u128))) };
     let value: u128 = transmute!(res);
     xor ^ value
-    }
 }
 
 #[allow(unused)]

